meta_configurator/src/schema/schemaPreprocessor.ts

```diff
@@ -69,55 +67,6 @@ function handleAllOfs(schema: JsonSchemaType) {
   return schema;
 }
 
-function convertTypesToOneOf(schema: JsonSchemaType) {
```

Comment:
Instead of converting type unions to oneOfs, I now just handle them completely serate. This sounds more work at first, but removes a lot of the complexity for cases where you would have oneOf and type unions and also where you have no explicit type (any type)